Loudspeaker_Explorer.md

```diff
@@ -88,7 +88,13 @@ import pathlib
 import re
 import textwrap
 
-running_in_colab = 'COLAB_GPU' in os.environ
+try:
+  import google.colab
+  IN_COLAB = True
+except:
+  IN_COLAB = False
+
+running_in_colab = IN_COLAB
```

Comment:
Why not set `running_in_colab` directly inside the `try`/`catch` block?

Reply:
Change made